src/Aspire.Hosting.Python/PythonProjectResourceBuilderExtensions.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Python;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Python applications to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class PythonProjectResourceBuilderExtensions
```

Comment:
It would be useful to add tests that *run* the python app too. See:
https://github.com/dotnet/aspire/blob/0af8505ff294c4b90ee8cccb5639bf73ec9077ba/tests/Aspire.Hosting.Tests/Node/NodeFunctionalTests.cs#L20-L32

https://github.com/dotnet/aspire/blob/0af8505ff294c4b90ee8cccb5639bf73ec9077ba/tests/Aspire.Hosting.Tests/TestProgramFixture.cs#L83-L92

https://github.com/dotnet/aspire/blob/0af8505ff294c4b90ee8cccb5639bf73ec9077ba/tests/testproject/TestProject.AppHost/TestProgram.cs#L64-L76